js/form.js

```diff
@@ -0,0 +1,58 @@
+import './data.js';
+const cardTemplate = document.querySelector('#card').content.querySelector('.popup');
+
+const getCardElement = (card) => {
+  const cardOfferCopy = cardTemplate.cloneNode(true);
+
+  const setContent = (selector, content) => {
+    const element = cardOfferCopy.querySelector(selector);
+    if (content) {
+      element.textContent = content;
+      element.src = content;
+    } else {
+      element.remove();
+    }
+  };
+
+  setContent('.popup__title',card.offer.title);
+  setContent('.popup__text--address', card.offer.address ? `${card.offer.address}`: false);
+  setContent('.popup__text--price',card.offer.price ? `${card.offer.price} ₽/ночь`: false);
+  const typesOffer = {
```

Comment:
Эту константу вверх файла и в верхнем регистре
```suggestion
  const OFFER_TYPES =
 
```